contrib/debugging-react-native.md

```diff
@@ -1,111 +0,0 @@
-
```

Comment:
Deleting this entirely, since almost all the information was outdated and would have to be re-tested anyway.